### ISSUE TITLE
Feature Request: Allow ignore file flag to be set on import code

### ISSUE THREAD

**Issue by davegillem** [bdkjones/CodeKit#406]
Often times when switching between branches and such the 'ignore issues' setting in the linked Files tab gets reset and turned on again. It would be great to have a flag that could be passed during the `@codekit-append` & `@codekit-prepend` line that would set it to be ignored for issues.

EX:
`@codekit-prepend --ignore 'bower_components/jquery/dist/jquery.min.js'`

**Comment by luxlogica** [comment]
This could be happening if you’re committing your Codekit config file. As you switch branches, the config file from the ‘other’ branch is loaded, and the settings you’ve just set seem to have ‘reverted’. If that is the case, then adding Codekit’s config file to your .gitignore should solve it.

**Comment by bdkjones** [comment]
The issue is that those ignore settings aren’t currently saved in the settings file for a very technical reason. It’s a shortcoming I need to address. I get that done for the next release.

**Comment by bdkjones** [comment]
I've debated the syntax for this. I didn't like `--ignore` because that looks like a CLI flag and the dashes don't really make sense in this context. I settled on this:

    //@codekit-prepend quiet 'someFile.js'

The positioning of `quiet` does not matter, so long as it's anywhere on the line and separated from other content by spaces. This will be equally valid:

    //@codekit-prepend 'someFile.js' quiet;

And if you have multiple files on the line, `quiet` will apply to all of them:

    //@codekit-prepend quiet 'someFile.js', 'otherFile.js'
